namelists_24.07.02/README

```diff
@@ -0,0 +1,2 @@
+Namelists valid for CABLE versions that include commit 30842390d87fda66286f5b81d79a4dbe2171f6e4 on 2024/07/02.
+Namelists valid for CABLE versions that do not include commit X on YYYY/MM/DD.
```

Comment:
My guess is that if we were to add a new version of namelist, we would need to edit line 2 (if that's the case, then it's fine for now)

Reply:
Yes, that's a placeholder, mainly to self-document to expected format because that's the start of it.